lang/funcs/simple/env_func.go

```diff
@@ -0,0 +1,76 @@
+// Mgmt
+// Copyright (C) 2013-2018+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// functions to look up variables in the mgmt server instance environment
+
+package simple // TODO: should this be in its own individual package?
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/purpleidea/mgmt/lang/types"
+)
+
+// getenv(str) str # get env or empty string if absent
+func getenv(input []types.Value) (types.Value, error) {
+	value := os.Getenv(input[0].Str())
+	return &types.StrValue{
+		V: fmt.Sprintf("%s", value),
```

Comment:
why is the fmt.Sprintf needed? this is a one-liner without the value var.
same below...